fiat/coincap_api.go

```diff
@@ -274,3 +325,47 @@ func (c *coinCapAPI) GetPrices(ctx context.Context, startTime,
 
 	return historicalRecords, nil
 }
+
+// coinCapRatesResponse holds the list of conversion rates returned by CoinCap.
+type coinCapRatesResponse struct {
+	Data []*coinCapRatePoint `json:"data"`
+}
+
+// coinCapRatePoint holds a currency symbol along with its conversion
+// rate to USD.
+type coinCapRatePoint struct {
```

Comment:
Say we're after the BTC/ZAR price. If we find our USD/ZAR rate in this response, then use it to convert our USD/BTC prices, wouldn't it be a case that a bitcoin price from last week will be converted to ZAR using today's price?